curtsies/__init__.py

```diff
@@ -1,8 +1,9 @@
 """Terminal-formatted strings"""
-__version__ = "0.3.1"
+__version__='0.3.2'
+
 
 from .window import FullscreenWindow, CursorAwareWindow
 from .input import Input
-from .termhelpers import Nonblocking, Cbreak, Termmode
+from .termhelpers import Nonblocking, Cbreak, Termmode 
```

Comment:
I'm going to remove the extra space here, FYI
```suggestion
from .termhelpers import Nonblocking, Cbreak, Termmode
```